frida_tools/tracer.py

```diff
@@ -62,7 +63,11 @@ def _initialize(self, parser, options, args):
             self._tracer = None
             self._targets = None
             self._profile = self._profile_builder.build()
-            self._quiet = options.quiet
+            self._quiet = options.quiet == True
```

Comment:
```suggestion
            self._quiet = options.quiet
```

Reply:
i introduced this true because quiet can be None, False and True and i want it to be a boolean not a quantum state

Reply:
but this is solved by using the default= in the options setup. solved

Reply:
The `optparse` module ensures that `True` will be stored if the user set the option. But it doesn't have a boolean type, at least not in the 2.7 version (didn't check 3.x), so the default is `None` if you don't specify one. So my thinking was that it is slightly simpler to do that there instead of normalizing it afterwards.